int32 r_mode	# CPUボードのモード切り替えスイッチの状態
int32 r_map	# コントロール操作時のマップ番号
int32 r_ctltype	# コントロールの種類(未接続=0、ゲームパッド=1、ProBo=2、RRC-T11=3)
bool r_torque	# サーボのトルクON/OFF ON=1/OFF=0
bool r_idle	# 現在アイドル中か否か アイドリング中=1 / アイドル以外のモーション再生中=0

